auth-web/src/views/auth/home/RequestNameView.vue

```diff
@@ -23,16 +23,14 @@
         </v-list-item>
         <!-- Panel Btns -->
         <div class="request-name-info-btns mt-5">
-          <name-request-button />
-          <p class="mt-5">Have an existing Name Request?
+          <name-request-button class="mr-2" :isInverse="true"/>
+          <learn-more-button :redirect-url="learnMoreUrl"/>
+        </div>
+        <p class="mt-5">Have an existing Name Request?
```

Comment:
Since paragraphs are block elements, move the text to the next line: it's clearer to the reader that the text is not inline with something else.